### DISCUSSION TITLE
Autoplay CPU player specific

### DISCUSSION THREAD

**Discussion by KennethLias** [TeamRizu/OutFox#268] [03 - Feature Requests]
Being able to play files while having Autoplay CPU on a specific player.

**Comment by shockdude** [comment]
Outfox already supports this.
Start a song with 2 players, go into Options, Additional Options, then set Player 2 AutoPlay to On. P2 will autoplay for the rest of the session.

Edit: oh wait do you mean "AutoPlayCPU" with judgments and stuff? Yeah I can see that.

**Reply by KennethLias** [reply]
Correct! 
I am aware About AutoPlay specific on player 2 but not Auto CPU

**Comment by jack5github** [comment]
To clarify, this user wants a specific user profile to always have player AutoPlay turned on for itself, which I can also get behind. I would really like to select a computer profile and immediately hop into a game without having the Main Options screen to get to the Additional Options screen every time I start the game.
